scheme/parsec-tpm/README.md

```diff
@@ -0,0 +1,30 @@
+# Endorsement Store Interface
+
+## Reference Value
+
+```json
+{
+  "scheme": "PARSEC_TPM",
+  "type": "REFERENCE_VALUE",
+  "attributes": {
+    "parsec-tpm.alg-id": 1,
```

Comment:
For consistency with Measurement Description (new change from https://github.com/veraison/services/pull/133 ) we should keep the Algorithm as a string.

Reply:
we need consistency between the evidence and provisioning side of the same scheme.